Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Collections.Specialized;
 using System.ComponentModel;
+using System.Linq;
```

Comment:
We've found that using Linq extensions in the Core and Platform code does not always produce the most optimized path of execution on devices, so we tend to avoid it. I don't believe that we've seen performance issues with `.Any` or `.Aggregate`, but it might be best to write a `foreach` in `IsTouchInDisabledButton` instead.